docs/source/development/cdf_guide.rst

```diff
@@ -0,0 +1,677 @@
+.. _cdf_guide:
+
+
+##############################
+Generating CDF Files
```

Comment:
Generating CDF files should be a section below CDF Files Introduction I think?

Maybe make the landing heading "CDF Guide" to correspond to what you called the file.